pkg/kotsadm/objects/kotsadm_objects.go

```diff
@@ -1061,16 +1016,7 @@ func KotsadmStatefulSet(deployOptions types.DeployOptions, size resource.Quantit
 									},
 								},
 							},
-							Resources: corev1.ResourceRequirements{
-								Limits: corev1.ResourceList{
-									"cpu":    resource.MustParse("1"),
-									"memory": resource.MustParse("2Gi"),
-								},
-								Requests: corev1.ResourceList{
-									"cpu":    resource.MustParse("100m"),
-									"memory": resource.MustParse("100Mi"),
-								},
-							},
+							Resources:       deployOptions.ResourceRequirements.KotsadmInit.ToCoreV1ResourceRequirements(),
```

Comment:
this will be change to init container resources. is there any reason this needed so high requests and limits?